tests/test_util.cpp

```diff
@@ -111,7 +111,7 @@ void initComms(int argc, char **argv, int *const commDims)
   initRand();
 
```

Comment:
Lots of lines changed here unnecessarily.  Any reason for this?  I realize there are some formatting fixes, but all these changes seem excessive given we like to preserve history.

Reply:
Looks like the _only_ changes were formatting, so I just copied the file from develop directly. Should hopefully cure this.